extensions/linear/src/components/ProjectList.tsx

```diff
@@ -25,7 +25,7 @@ export default function ProjectList() {
       return [];
     }
 
-    if (roadmap === "") {
+    if (!roadmap || !roadmaps || roadmaps.length < 1) {
```

Comment:
The issue was a bit subtle:
- `roadmap` can be `undefined`, so this string equality check would be evaluated to false, even if there was no `roadmap` to filter the projects.

I also added a check for the roadmaps. It seems conceptually correct to check if there are roadmaps available, instead of only relying on the `roadmap` filter being implicitly invalid if there aren't any roadmaps.